lib/rubocop/options.rb

```diff
@@ -362,16 +362,19 @@ def validate_parallel
 
     def validate_parallel_with_combo_option
```

Comment:
Maybe we should rename this method now?

Reply:
I had the same thought. I couldn't think of great name for what this is doing. I think this is the first time we're disabling incompatible configs so maybe a new method entry point? Maybe something like `fix_invalid_parallel_combos` or `disable_invalid_parallel_option`?